### DISCUSSION TITLE
How to list all the sub-task of one task

### DISCUSSION THREAD

**Discussion by caenrique** [ankitpokhrel/jira-cli#203] [Q&A] [answered]
I want to find all the subtask of a given task, and I'm not sure how to do it. 
Is there a mapping to do it in the interactive ui, or a command which I can provide with a id ?

**Comment by ankitpokhrel** [comment, marked as answer]
Hi @caenrique, you can use `-P/--parent` flag for that. 

```
jira issue list -P ISS-123
```

However this is not yet available in the release. I will make a new release in few days.

For now, if you wish to use this feature, you can [compile from source](https://github.com/ankitpokhrel/jira-cli#development) or if you are using homebrew you can build binary from `HEAD` (requires `make`, `git` and `go`)

```sh
brew uninstall jira-cli
brew install jira-cli --HEAD
```

**Reply by caenrique** [reply]
Thanks!
